tests/console/clamav.pm

```diff
@@ -28,7 +28,7 @@ sub scan_and_parse {
     my $cmd = shift;
     my $log_file = "$cmd.log";
 
-    script_run "$cmd -i --log=$log_file eicar_test_files", 300;
+    script_run "$cmd -i --log=$log_file eicar_test_files", 1800;
```

Comment:
why did you change this timeout? As far as I can see from the progress ticket the fail happens when you try to start clamd service.

Reply:
Yes but when I made my tests, it ended up that this gave a timeout too. So I went ahead and fixed all the timeouts.

Reply:
what was the percentage of failure here? still I would suggest that you try a smaller timeout than 1800s